pkg/apis/pipeline/v1alpha1/taskrun_types.go

```diff
@@ -103,6 +104,9 @@ type TaskRunStatus struct {
 	// +optional
 	Conditions duckv1alpha1.Conditions `json:"conditions,omitempty"`
 
+	// In #107 should be updated to hold the location logs have been uploaded to
+	Results Results `json:"results"`
```

Comment:
same comment as above, the results field is optional in the taskRun, but the field in the status is not